sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -64,6 +85,53 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
         val tableIdent = extractTableIdent(nameParts)
         RefreshTable(tableIdent)
 
+      case Token("TOK_CREATEDATABASE", Token(databaseName, Nil) :: createDatabaseArgs) =>
+        val Seq(
+          allowExisting,
```

Comment:
Isn't `allowExists` the exact opposite of `IF NOT EXISTS`? I am asking because I have a similar case in my PR.